pkg/api/v1/resource/helpers.go

```diff
@@ -136,6 +136,11 @@ func GetResourceRequestQuantity(pod *v1.Pod, resourceName v1.ResourceName) resou
 		return requestQuantity
 	}
 
+	// pod swap limitation is not supported and this is only for scheduling
+	if resourceName == v1.ResourceSwap {
+		return requestQuantity
```

Comment:
This went through the default fallthrough case which means we get `Format: resource.DecimalSI`, but I imagine we would want `Format: resource.BinarySI`?

Reply:
Still needs addressing. Please update L128 to include v1.ResourceSwap rather than falling through to the default.